src/app/runtime_genesis_ledger/runtime_genesis_ledger.ml

```diff
@@ -97,7 +95,7 @@ let load_config_exn ~logger config_file =
   in
   if
     Option.(
-      is_some config.daemon || is_some config.genesis
```

Comment:
I had to remove this check as now genesis node is added automatically when Runtime_config is loading json

Reply:
Sorry, not following this one. Is `genesis` being added by `mina ledger test generate-accounts`?

Comment is an old one, maybe we could just drop this change?